WtWSMS/common/culture/creation_names/BP_ck2_custom_names.txt

```diff
@@ -0,0 +1,238 @@
+﻿#The most glorious of all cultures, East Celtic
+east_celtic = {
+	trigger = {
+		has_valid_east_celtic_ancestors_trigger = {
+			CULTURE_1 = scope:culture
+			CULTURE_2 = scope:other_culture
+		}
+	}
+	hybrid = yes
+}
+##### THIS SECTION BASED ON CK2/WtWSMS/events/WtWSMS_culture_conversion_events.txt
+#Note that since these *aren't* county events in CK3, we don't do the religion checks wrt counties
+#ROM.1000: Skipped (Duplicate French Creation)
+#ROM.1100: Skipped (Duplicate Occitan Creation)
+#ROM.1150: Skipped (Visigoth back to Romano-Gallic)
+gothi = { #ROM.2000; Gothi
+	trigger = {
+		root = {
+			culture = culture:visigothic
+			faith.religious_head = faith:nicene.religious_head
+			capital_county = { any_county_province = { geographical_region = world_europe_west_iberia } }
+		}
+		scope:other_culture = { this = culture:romano_hispanic }
+	}
+	hybrid = yes
+}
+#ROM.2001: Skipped (Continential Celtic Province to Romano-Hispanic)
+#ROM.2048: Skipped (Gaulish Province to Romano-Gallic)
+#ROM.2100: Skipped (Iberian Clusterfuck Part 1: Catalans)
+#ROM.2200: Skipped (Iberian Clusterfuck Part 2: Portuguese)
+#ROM.2300: Skipped (Iberian Clusterfuck Part 3: Castilian)
+#ROM.2400: Skipped (Iberian Clusterfuck Part 4: Leonese)
+#ROM.2500: Skipped (Iberian Clusterfuck Part 5: Andalusian)
+#ROM.3000: Skipped (Latin to Italian)
+#CELT.1000: Skipped (Pictish to Scottish)
+breton = { #CELT.1100: Amorican to Breton
+	trigger = {
+		root = {
+			OR = {
+				culture = culture:briton
+				culture = culture:romano_british
+			}
+			capital_county = { any_county_province = { geographical_region = custom_breton } }
+		}
+		scope:other_culture = culture:armorican
+	}
+	hybrid = yes
+}
+##### THIS SECTION BASED ON CK2/WtWSMS/events/WtWSMS_melting_pot_evnts_events.txt
```

Comment:
What about religious and government conditions which slowed down the process and made it feel more progressive?

Reply:
Given the mechanics in CK3, large cultures are prone to hybridization/divergence. I've added some parameters to Romance/Hellenic cultures that will make it difficult to diverge cultures so long as they are consolidated into a single polity, but I still expect them to hybridize after migration or diverge if the culture gets split among different polities.

Given that linguistic/heritage drift is now mechanically separate from culture drift, I think it's alright to allow cultures to arise earlier and have the linguistic changes follow later.

Reply:
> Given the mechanics in CK3, large cultures are prone to hybridization/divergence. I've added some parameters to Romance/Hellenic cultures that will make it difficult to diverge cultures so long as they are consolidated into a single polity, but I still expect them to hybridize after migration or diverge if the culture gets split among different polities.
> 
> Given that linguistic/heritage drift is now mechanically separate from culture drift, I think it's alright to allow cultures to arise earlier and have the linguistic changes follow later.

Maybe, but how early? It already appeared early in CK2. And I think there is still something to be said about religion and government, if you don't share the same faith and do not share the same administration, not much will bring you together into a common culture.

Reply:
As I mentioned earlier, I'll do a refactor on the event firing mechanics in on_actions and the event structure itself. Then we can revisit.